src/main/java/com/emc/ecs/tool/BucketWipe.java

```diff
@@ -126,6 +142,9 @@ private static Options options() {
                 .desc("instead of listing bucket, delete objects matched in source file key list").build());
         options.addOption(Option.builder().longOpt("delete-mpus").desc("incomplete MPUs will prevent the " +
                 "bucket from being deleted. use this option to clean up all incomplete MPUs").build());
+        options.addOption(Option.builder("y").longOpt("accept-disclaimer").argName("accept-disclaimer")
```

Comment:
this should not have an argument (no `argName`)

Reply:
fixed